calico_node/tests/st/utils/docker_host.py

```diff
@@ -670,8 +676,10 @@ def _exec_calicoctl(self, action, data):
     def log_extra_diags(self):
         # Run a set of commands to trace ip routes, iptables and ipsets.
         self.execute("ip route", raise_exception_on_failure=False)
-        self.execute("iptables-save", raise_exception_on_failure=False)
-        self.execute("ip6tables-save", raise_exception_on_failure=False)
+        self.execute("iptables-save -c", raise_exception_on_failure=False)
+        self.execute("ip6tables-save -c", raise_exception_on_failure=False)
         self.execute("ipset save", raise_exception_on_failure=False)
         self.execute("ps", raise_exception_on_failure=False)
-        self.execute("cat /etc/bird/bird.conf", raise_exception_on_failure=False)
+        self.execute("docker logs calico-node", raise_exception_on_failure=False)
```

Comment:
Why did you remove dumping the bird.conf? I think it should remain unless you know that it isn't useful anymore.

Reply:
The line as was did not work for two reasons:

1. The BIRD config is inside the calico-node container, not directly on the host.

2. The BIRD config is not at that path, even within the container.

I think a correct line would be something like
```
self.execute("docker exec calico-node cat /etc/calico/confd/bird.cfg", raise_exception_on_failure=False)
```
but I don't currently have reason to need that.
￼